com.chenyifaer.FaFaRunner.yml

```diff
@@ -0,0 +1,74 @@
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=dri
+  - --share=network
+
+modules:
+  - name: fafarunner
+    buildsystem: simple
+    build-commands:
+      - bsdtar -Oxf fafarunner.deb data.tar.zst | bsdtar -xf -
+      - install -Dm755 usr/share/fafarunner/fafarunner /app/bin/fafarunner
+      - cp -r usr/share/fafarunner/data usr/share/fafarunner/lib /app/bin/
+      - install -Dm644 com.chenyifaer.FaFaRunner.desktop /app/share/applications/com.chenyifaer.FaFaRunner.desktop
+      - install -Dm644 usr/share/icons/hicolor/128x128/apps/fafarunner.png /app/share/icons/hicolor/128x128/apps/com.chenyifaer.FaFaRunner.png
+      - install -Dm644 usr/share/icons/hicolor/256x256/apps/fafarunner.png /app/share/icons/hicolor/256x256/apps/com.chenyifaer.FaFaRunner.png
+      - install -Dm644 com.chenyifaer.FaFaRunner.metainfo.xml /app/share/appdata/com.chenyifaer.FaFaRunner.metainfo.xml
+    sources:
+      - type: file
+        only-arches:
+          - x86_64
+        url: https://github.com/fafarunner/fafarunner/releases/download/v1.0.0+355/fafarunner_1.0.0+355_linux_amd64.deb
+        dest-filename: fafarunner.deb
+        sha256: 8444a0cfd7ad24e2cc5c5f9a096538a8a55a2c7ae8adb7d8e2dfd3c9320e0518
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/fafarunner/fafarunner/releases/latest
+          version-query: .tag_name | sub("v";"")
+          url-query: .assets[] | select(.name=="fafarunner_" + $version + "_linux_amd64.deb") | .browser_download_url
+
+      - type: file
+        only-arches:
+          - aarch64
+        url: https://github.com/fafarunner/fafarunner/releases/download/v1.0.0+355/fafarunner_1.0.0+355_linux_aarch64.deb
+        dest-filename: fafarunner.deb
+        sha256: 84dbd660fe4c3e76091470314cea23b843a2fc8551fb2656275e972f7decfba5
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/fafarunner/fafarunner/releases/latest
+          version-query: .tag_name | sub("v";"")
+          url-query: .assets[] | select(.name=="fafarunner_" + $version + "_linux_aarch64.deb") | .browser_download_url
+
+      - type: file
+        url: https://raw.githubusercontent.com/fafarunner/fafarunner/v1.0.0+355/linux/shared/com.chenyifaer.FaFaRunner.desktop
+        dest-filename: com.chenyifaer.FaFaRunner.desktop
+        sha256: eefde8dc8a139ae8102a43bebaea7f2334076cbf685ae1b5243f0b3496cff252
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/fafarunner/fafarunner/releases/latest
+          version-query: .tag_name
+          url-query: "\"https://raw.githubusercontent.com/fafarunner/fafarunner/\" + $version + \"/linux/shared/com.chenyifaer.FaFaRunner.desktop\""
+
+      - type: file
+        url: https://raw.githubusercontent.com/fafarunner/fafarunner/v1.0.0+355/linux/shared/com.chenyifaer.FaFaRunner.metainfo.xml
```

Comment:
this is not in the .deb? it should be.

Reply:
This file is universal and is also used by other Linux format packages.

Reply:
then it should be in the .deb